spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -94,6 +94,9 @@ spring:
         dependency-fetch:
           enabled: false
         dependencies:
+          spring-cloud-dataflow-server:
+            name: Spring Cloud Dataflow
+            version: "@project.version@"
```

Comment:
Should this be `@dataflow.version@` ?

Reply:
It will only build `spring-cloud-dataflow-server-core` in the scope of `spring-cloud-dataflow` and the project.version will be the same as `dataflow.version`

Reply:
As long as you have tested both OSS and PRO - this is fine.